In [25]:
import pandas as pd
from math import sqrt


Null Hypothesis: The average time for X8 to complete the original route is the same or less than the average time to complete the new route, significant level p = 0.05.

Alternative Hypothesis: The average time for X8 to complete the original route is more than the average time to complete the new route.
  
      
$H_0$: $\mu_0$ <= $\mu_1$

$H_1$: $\mu_0$ > $\mu_1$

$\alpha$ = 0.05


In [26]:
df = pd.read_fwf('times.txt', names=['Time'])

In [27]:
new_mean = df['Time'].mean()
new_std = df['Time'].std()
n = df['Time'].count()
ori_mean = 36
ori_std = 6

In [28]:
print(new_mean)
print(new_std)

34.4661616883
7.13728013375


In [29]:
z = (ori_mean - new_mean)/(ori_std/sqrt(n))

In [30]:
print(z)

2.55639718617


2.556 sigma (z = 2.556) indicates an out of 95% probability, which means the probability to obtain a sample mean of 34.466 is less than 5%. 

However, $\alpha=0.05$ corresponds to 95% inclusion, which means the probability to obtain a certain sample mean should be equal or more than 5%.

Based on the information, we can conclude: P value < 0.05. 

So, we have to reject the null hypothesis, and conclude that the average time for X8 to complete the original route is significantly more than the average time to complete the new route. The implementation of an alternative bus route for bus line X8 can improve circulation.